<launch>

    <arg name="rate" default="10.0"/>

    <node pkg="centroidal_planner"
         name="centauro_DEMO" type="centauro_DEMO.py" output="screen">
        <param name="mu" value="0.3"/>
        <rosparam param="com0_offset"> [0.0, 0.0, -0.05] </rosparam>
        <rosparam param="tau_ext"> [1200.0, 0.0, 0.0, 0.0, 0.0, 0.0] </rosparam>
        <param name="F_thr" value="200.0"/>
        <param name="surface_reach_thr" value="20.0"/>
    </node>

    
</launch>
